pom.xml

```diff
@@ -176,4 +220,147 @@
+                                            <hide>
+                                                <categories>
+                                                    <category>merge</category>
+                                                    <category>tasks</category>
+                                                    <category>build</category>
+                                                </categories>
+                                                <contributors>
+                                                    <contributor>GitHub</contributor>
+                                                </contributors>
+                                            </hide>
+                                            <contributors>
+                                                <format>- {{contributorName}}</format>
+                                            </contributors>
+                                        </changelog>
+                                        <draft>true</draft>
+                                        <tagName>${project.version}</tagName>
+                                        <releaseName>Release ${project.version}</releaseName>
+                                        <prerelease>
+                                            <pattern>.*\.(alpha|beta|rc)\d+</pattern>
+                                        </prerelease>
+                                    </github>
+                                </release>
+                                <deploy>
+                                    <maven>
+                                        <artifactory>
+                                            <app>
+                                                <active>${release.jfrog.active}</active>
+                                                <url>https://mcollovati.jfrog.io/artifactory/jreleaser-tests</url>
+                                                <stagingRepositories>target/staging-deploy</stagingRepositories>
+                                                <applyMavenCentralRules>true</applyMavenCentralRules>
+                                            </app>
+                                        </artifactory>
+                                        <nexus2>
+                                            <maven-central>
+                                                <active>${release.mavencentral.active}</active>
+                                                <url>https://oss.sonatype.org/service/local</url>
+                                                <snapshotUrl>https://s01.oss.sonatype.org/content/repositories/snapshots/</snapshotUrl>
+                                                <closeRepository>false</closeRepository>
+                                                <releaseRepository>false</releaseRepository>
+                                                <stagingRepositories>target/staging-deploy</stagingRepositories>
```

Comment:
Correct repo?

Reply:
Yes. Artifacts are staged locally before pushing on Maven central and/or jfrog

Reply:
Is sonatype local? Don't understand what this does

Reply:
Basically we first do a deploy to a local folder, the jreleaser uploads artifacts to nexus from that folder